src/cleo/_utils.py

```diff
@@ -51,12 +50,12 @@ def find_similar_names(name: str, names: list[str]) -> list[str]:
     """
     Finds names similar to a given command name.
     """
-    threshold = 1e3
+    threshold = 0.4
```

Comment:
Can you somehow justify this new threshold? Maybe move it to a global constant?

Reply:
difflib return values from a different range than rapidfuzz.